test-containers/HelmChartTestWeb.Tests/Fixtures/DbServerFixture.cs

```diff
@@ -20,20 +20,28 @@ public class DbServerFixture : IAsyncLifetime, IDisposable
 
         private readonly TestcontainersContainer _testcontainersBuilder;
         public ApplicationDbContext DbContext { get; }
+        public int MsSqlServerContainerPublishedPort { get; } = new Random().Next(1025, 65535);
```

Comment:
I think TestContainer has a random port selector that will find an unused port on the server

Reply:
Done. You were right and it has a respective API for that  as you mentioned.